<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pitchDialog</class>
 <widget class="QDialog" name="pitchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pitch Rate Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Input</string>
       </property>
       <property name="configSection" stdset="0">
        <string notr="true">pitchRateCalc</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="BtVolumeLabel" name="label_vol">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Wort Volume</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_vol</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="BtVolumeEdit" name="lineEdit_vol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Volume of wort</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">wortVol</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="BtDensityEdit" name="lineEdit_OG">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Starting gravity of the wort</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">og</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_pitchRate_2">
          <property name="toolTip">
           <string>For ales, 0.75-1. For lagers, 1.5-2.</string>
          </property>
          <property name="text">
           <string>Pitch Rate (M cells)/(mL*P)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="slider_pitchRate">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Desired pitch rate</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_pitchRate">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_vialsPitched">
          <property name="text">
           <string>Vials/Smack Packs Pitched</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBox_VialsPitched">
          <property name="toolTip">
           <string>Number of vials/smack packs added to starter</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_aerationMethod">
          <property name="text">
           <string>Aeration Method</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBox_AerationMethod">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Aeration method</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>O2 At Start</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stir Plate</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="BtDateLabel" name="label_productionDate">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Yeast Production Date</string>
          </property>
          <property name="buddy">
           <cstring>dateEdit_ProductionDate</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDateEdit" name="dateEdit_ProductionDate">
          <property name="toolTip">
           <string>Production date (Best By date less three months)</string>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::MonthSection</enum>
          </property>
          <property name="displayFormat">
           <string>MM/dd/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">productionDate</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_viability">
          <property name="text">
           <string>Yeast Viability %</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="spinBox_Viability">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Estimated viability of the yeast</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_CalculateViability">
          <property name="text">
           <string>Calculate Viability From Date</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="BtDensityLabel" name="label_og">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Original gravity</string>
          </property>
          <property name="text">
           <string>OG</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_OG</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Output</string>
       </property>
       <property name="configSection" stdset="0">
        <string notr="true">pitchRateCalc</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_cells">
          <property name="text">
           <string>Billions of Yeast Cells Required</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_cells</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="BtGenericEdit" name="lineEdit_cells">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>How much yeast you will need</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_vials">
          <property name="text">
           <string>Vials/Smack Packs w/o Starter</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_vials</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="BtGenericEdit" name="lineEdit_vials">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>How many smack packs or vials required to reach pitch rate</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="BtMassLabel" name="label_yeast">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Dry Yeast</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_yeast</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="BtMassEdit" name="lineEdit_yeast">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Amount of dry yeast needed to reach pitch rate</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">yeast</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="BtVolumeLabel" name="label_starterVol">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Starter Volume</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_starterVol</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="BtVolumeEdit" name="lineEdit_starterVol">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Starter size to reach pitch rate</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">starterVol</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtGenericEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>BtDensityEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtDensityLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtMassLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
    <slot>popContextMenu(QPoint)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtMassEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <signal>textModified()</signal>
    <slot>lineChanged()</slot>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtDateLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_vol</tabstop>
  <tabstop>lineEdit_OG</tabstop>
  <tabstop>slider_pitchRate</tabstop>
  <tabstop>spinBox_VialsPitched</tabstop>
  <tabstop>comboBox_AerationMethod</tabstop>
  <tabstop>dateEdit_ProductionDate</tabstop>
  <tabstop>spinBox_Viability</tabstop>
  <tabstop>checkBox_CalculateViability</tabstop>
  <tabstop>lineEdit_cells</tabstop>
  <tabstop>lineEdit_vials</tabstop>
  <tabstop>lineEdit_yeast</tabstop>
  <tabstop>lineEdit_starterVol</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>label_vol</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_vol</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_og</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_OG</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_starterVol</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_starterVol</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_yeast</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_yeast</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
